src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
    ...
 ----
 
+=== Deploy ProActive Nodes dynamically via other schedulers (PBS, SLURM, ...)
+
+This functionality is also called `Meta-Scheduling`.
+
+If an existing cluster is available in your organization and this cluster is managed by a native scheduler such as *SLURM*, *LSF* or *PBS*, you may want to execute transparently on this cluster ProActive <<__glossary_workflow,*Workflows*>> and <<_glossary_task,*Tasks*>>.
+
+As ProActive Tasks can be executed only on <<_glossary_proactive_node,*ProActive Nodes*>>, it is necessary to deploy ProActive Nodes on the cluster.
+
+Dynamic deployment of ProActive Nodes on a cluster to execute ProActive Tasks is handled by defining a _Native Scheduler <<_glossary_node_source,Node Source>>_ containing three components:
+
+ * The `Native Scheduler Infrastructure`: a <<_glossary_node_source_infrastructure,*Node Source Infrastructure*>> which allows to interact with a native scheduler to deploy ProActive Nodes.
```

Comment:
"`which allows to interact with a native scheduler`" maybe it is more accurate to say ` with a native cluster`? If yes we can define the term `native cluster` at the beginning of the section.

Reply:
I try to make the distinction between the cluster (infrastructure), and the native scheduler (the software which manages the infrastructure)